tests/python/test_custom_struct.py

```diff
@@ -315,3 +315,24 @@ def test():
         assert b.b == 4
 
     test()
+
+
+@ti.test(debug=True)
+def test_copy_struct_field_element_to_taichi_scope():
```

Comment:
OOC: why do we need to set `debug=True` here? ;)

Reply:
Because assertions in kernels are enabled only when `debug=True` :-)